framework/doc/content/source/transfers/MultiAppInterpolationTransfer.md

```diff
@@ -2,7 +2,16 @@
 
 The MultiAppInterpolationTransfer transfers the source variable to the nearest node on the
 target mesh using mesh interpolation, including the ability to utilize the displaced
-configuration for either or both the source and target.
+configuration for either or both the source and target.  
+
+Nodal transfers using this interpolation scheme is best suited for the interpolation of point cloud data in the source mesh onto a target mesh.  
+The [MultiAppMeshFunctionTransfer](/MultiAppMeshFunctionTransfer.md) may be a better choice for nodal transfers between two meshes because element data will be used in the transfer and the transfer is not dependent on the number of points used in the transfer.  
+Using it for mesh to mesh nodal transfers of data can lead to a parallel inconsistency that arises when the point being interpolated to on the
+target mesh is exactly the same distance away from more than `num_points` on the source mesh.  
+This inconsistency goes away by increasing `num_points`.   
+For unstructured meshes, this parallel inconsistency should not be an issue.  
```

Comment:
I d remove this line. It could still happen. And a lot of UM are a little structured near boundaries

Reply:
I didn't want to freak people out.  I think this transfer usually works.  I can make "should" itallic.